templates/profile.html

```diff
@@ -98,10 +99,28 @@ <h1>{{ participant.username }}</h1>
                         <a class="btn btn-donating btn-lg" href="{{ donate_path }}">{{
                             _("Donating ({amount})", amount=tip.amount)
                         }}</a>
-                    % else
-                        <a class="btn btn-donate btn-lg" href="{{ donate_path }}">
-                            {{ _("Donate") }}
-                        </a>
+                    % else                    
+                        <span title={{ accepts_summary }}><a class="btn btn-donate btn-lg" href="{{ donate_path }}">
```

Comment:
You forgot the quotes around `{{ accepts_summary }}`, it should be `title="{{ accepts_summary }}"`.

Reply:
Brilliant. I've made that mistake before!